index.html

```diff
@@ -50,7 +50,21 @@ <h5 id="equation-status"></h5>
 
         <!-- Content Section -->
         <div class="nine columns">
-          <canvas id="canvas"></canvas>    
+          <canvas id="canvas"></canvas>  
+          <table >
+            <thead>
+              <tr>
+                <th style="width: 20%;">A</th>
+                <th style="width: 20%;">B</th>
+                <th style="width: 20%;">W</th>
+                <th style="width: 30%;">F(A).F(W)</th>
+                <th style="width: 30%;">F(A).F(W)</th>
+              </tr>
+            </thead>
+            <tbody id="myTable">
```

Comment:
id naming convention used is to be id="my-table"
and the name is not relevant use say 'steps-table' instead.